templates/records/blocks/record_links_block.html

```diff
@@ -1,23 +1,24 @@
 {% load wagtailimages_tags %}
-<div class="recordlinks__wrapper">
-    <ul class="recordlinks__list">
+<div class="record-links__wrapper">
+    <ul class="record-links__list">
         {% for item in value.items %}
-            {% with item.record.url as url %}
-                <li class="recordlinks__item">
-                    {% if item.thumbnail_image %}
-                        <div class="recordlinks__item__thumbnail">
-                            <a href="{{ url }}">
-                                {% image item.thumbnail_image fill-200x200 alt="" class="recordlinks__thumbnail-image" %}
-                            </a>
+                <li>
+                    <a href="{{ item.record.url }}">
```

Comment:
After a discussion with Will & Nick last week we've agreed to limit the links on cards to the title (or title and image) only. I think for this element keeping it to the title alone is sufficient as the image is quite small). So we'll need to move the `<a>` tag to inside the title please

Reply:
Hi @janehughes okay, thank you!
What link interaction are we having when hovering over link text? Every link seems to be different at the minute as it's all changing so I'm not sure what to copy :)

Reply:
Nick's very helpfully included all the link interaction styles in the design system - you can find this component here https://www.figma.com/file/uo8YY98lLws8E1HEfpdn5X/TNA-Etna---Design-system?type=design&node-id=2846-40842&t=dA9vGMTUz1iAojNv-0 :) If you skim through the other components though you can see general link styles follow the same pattern. Essentially remove underline on hover and use the blue colour. Focus should stay as the default (thick blue border)